engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaBackendIsCopyingCachedOutputsSpec.scala

```diff
@@ -96,17 +98,17 @@ class EjeaBackendIsCopyingCachedOutputsSpec extends EngineJobExecutionActorSpec
         }
 
         RestartOrExecuteCommandTuples foreach { case RestartOrExecuteCommandTuple(operationName, restarting, expectedMessage) =>
-          s"$operationName the job immediately when it gets a failure result, and it was going to receive $hashComboName, if call caching is $mode" in {
+          s"invalidate a call for caching if backend coping failed when start mode is $operationName, and it was going to receive $hashComboName, if call caching is $mode" in {
```

Comment:
This no longer seems to depend on the `RestartOrExecuteCommandTuples` - the result is the same either way. (I think that now only the `EjeaInvalidatingCacheEntrySpec ` needs to check that it works right depending on the restart mode)